index.js

```diff
@@ -401,6 +401,10 @@ class Squeakquel extends Datastore {
         }
 
         if (config.search && config.search.field && config.search.keyword) {
+            const searchKey = this.client.getDialect() === 'postgres'
+                ? { [Sequelize.Op.ilike]: config.search.keyword }
+                : { [Sequelize.Op.like]: config.search.keyword };
```

Comment:
how about mysql/mariaDB ? can't we do case insensitive for them ?

Reply:
Seeing this: https://stackoverflow.com/questions/38154668/mariadb-mysql-querys-are-case-sensitive

Reply:
To make it generic for other cases we would need convert to lower case for both and compare.